### PR TITLE
Migrate `RecentPatientsScreen` to a fragment

### PR DESCRIPTION
**Pull request by msasikanth**

### CODE REVIEW

**Review by Honey14 — APPROVED** [review]
LGTM

File: app/src/main/res/layout/recent_patients_screen.xml [hunk 1]
```diff
@@ -1,14 +1,15 @@
 <?xml version="1.0" encoding="utf-8"?>
-<org.simple.clinic.recentpatient.RecentPatientsScreen xmlns:android="http://schemas.android.com/apk/res/android"
```
[the remark this answers is not in the document]
Comment (msasikanth) [reply app/src/main/res/layout/recent_patients_screen.xml:2]: I don't see it 😅

[the remark this answers is not in the document]
Comment (Honey14) [reply app/src/main/res/layout/recent_patients_screen.xml:2]: You'll be able to see in the First commit message

[the remark this answers is not in the document]
Comment (msasikanth) [reply app/src/main/res/layout/recent_patients_screen.xml:2]: Ah, I was looking at the 2nd commit 👍🏾

**Pull request merged**